deploy/genesis/docker-compose-integration.yml

```diff
@@ -0,0 +1,118 @@
+version: '3'
```

Comment:
Same again. Move this file under `./test/*` somewhere.

Reply:
Moved to test/integration

Reply:
Turns out this is causing problems, since the scripts have hidden dependencies on the directory name.  Taking a look.